ui/components/ui/account-list/account-list.js

```diff
@@ -73,67 +89,118 @@ const AccountList = ({
                 className="choose-account-list__account"
                 ref={isSelectedAccount ? selectedAccountScrollRef : null}
+                backgroundColor={
+                  isSelectedAccount
+                    ? Color.primaryMuted
+                    : BackgroundColor.backgroundDefault
+                }
               >
-                <div className="choose-account-list__account-info-wrapper">
-                  <CheckBox
-                    className="choose-account-list__list-check-box"
-                    checked={isSelectedAccount}
-                  />
-                  <Identicon diameter={34} address={address} />
-                  <div className="choose-account-list__account__info">
-                    <div className="choose-account-list__account__label">
-                      {addressLabel}
-                    </div>
-                    <UserPreferencedCurrencyDisplay
-                      className="choose-account-list__account__balance"
-                      type={PRIMARY}
-                      value={balance}
-                      style={{ color: 'var(--color-text-alternative)' }}
-                      suffix={nativeCurrency}
-                    />
-                  </div>
-                </div>
-                {addressLastConnectedMap[address] ? (
-                  <Tooltip
-                    title={`${t('lastConnected')} ${
-                      addressLastConnectedMap[address]
-                    }`}
+                <Box
+                  display={Display.Flex}
+                  width={BlockSize.Full}
+                  alignItems={AlignItems.center}
+                >
+                  <Checkbox isChecked={isSelectedAccount} />
+                  <Box marginLeft={2}>
+                    <Identicon diameter={34} address={address} />
+                  </Box>
+                  <Box
+                    display={Display.Flex}
+                    justifyContent={JustifyContent.spaceBetween}
+                    width={BlockSize.Full}
+                    paddingLeft={3}
+                    style={{
+                      minWidth: 0,
+                    }}
                   >
-                    <Icon
-                      name={IconName.Info}
-                      color={IconColor.iconMuted}
-                      className="info-circle"
-                      marginInlineStart={2}
-                    />
-                  </Tooltip>
-                ) : null}
-              </div>
+                    <Box
+                      display={Display.Flex}
+                      flexDirection={FlexDirection.Column}
+                      width={BlockSize.Full}
+                    >
+                      <Text
+                        variant={TextVariant.bodyLgMedium}
+                        style={{
+                          textWrap: 'nowrap',
+                        }}
+                        ellipsis
+                      >
+                        {addressLabel}
```

Comment:
It is some kind of double ellipsis, that's true. But, that's the data returned for the account name formatted like that and used in previous implementation. We're adding one more ellipsis at the end in case that the content does not fit. This is found as the best solution probably, when discussed with @eriknson

Reply:
Ye, this is intended for now but the accounts team will revisit this component later. The first `...` is for the middle truncation and the latter `...` is because it's too long in total

Reply:
@eriknson to follow up with ticket for this requirement. I'm resolving this conversation on this PR for now.